scripts/resource/auto_busybox.rc

```diff
@@ -0,0 +1,59 @@
+# portscan.rc
```

Comment:
After reviewing this resource file (auto_busybox.rc), i feel like it is something like your personal solution when working with some busybox devices, maybe for testing the modules in this PR. But I'm not sure if it worths as generic automation resource. I can bet people having problems, for example, because of trying to run the resource file on a long pentest, with a big bunch of sessions opened.  Because of it, I think it would have more sense if you publish it on your personal clone repository, than merging this resource file into rapid7/master